### PR TITLE
remove drop function test

### PR DESCRIPTION
**Pull request by conorbros**
Main is failing with this error: 
```
2022-08-25T12:24:54.7233573Z thread 'cassandra_int_tests::test_cluster_multi_rack' panicked at 'Expected the error to contain 'Unknown function 'test_function_keyspace.my_function'' but it did not and was instead 'Cassandra detailed error SERVER_INVALID_QUERY: Invalid number of arguments in call to function
```

I can't find any mention of functions in the schema version agreement docs so that leads me to believe that awaiting for schema agreement does not include functions.

Decided to remove this test rather than implement something to wait for changes to functions to propagate throughout the cluster since it's not testing any particular edge case.

### CODE REVIEW

File: shotover-proxy/tests/cassandra_int_tests/functions.rs [hunk 1]
```diff
@@ -1,19 +1,5 @@
...
 
-async fn drop_function(session: &CassandraConnection) {
-    assert_query_result(
-        session,
-        "SELECT test_function_keyspace.my_function(x, y) FROM test_function_keyspace.test_function_table WHERE id=1;",
-        &[&[ResultValue::Int(4)]]
-    ).await;
-    run_query(session, "DROP FUNCTION test_function_keyspace.my_function").await;
-
-    session.execute_expect_err_contains(
```
Comment (rukai) [inline shotover-proxy/tests/cassandra_int_tests/functions.rs:11]: What if we deleted just this line, so we still get to check that "DROP FUNCTION" doesnt blow up somehow?

Comment (conorbros) [reply shotover-proxy/tests/cassandra_int_tests/functions.rs:11]: Sure might as well

**Review by rukai — APPROVED** [review]
LGTM!

**Review by benbromhead — APPROVED** [review]
